Building package tiff:x86-uwp failed with: BUILD_FAILED

C:\Users\ILYVOI\Git\vcpkg>.\vcpkg install opencv[contrib]:x86-uwp --recurse
The following packages will be built and installed:
  * libpng[core]:x86-uwp
    opencv[contrib,core,eigen,flann,jpeg,opengl,png,tiff]:x86-uwp
  * opengl[core]:x86-uwp
  * tiff[core]:x86-uwp
Additional packages (*) will be modified to complete this operation.
Starting package 1/4: tiff:x86-uwp
Building package tiff[core]:x86-uwp...
-- Using cached C:/Users/ILYVOI/Git/vcpkg/downloads/tiff-4.0.10.tar.gz
-- Using source at C:/Users/ILYVOI/Git/vcpkg/buildtrees/tiff/src/4.0.10-208d032666
-- Configuring x86-uwp
-- Building x86-uwp-dbg
CMake Error at scripts/cmake/vcpkg_build_cmake.cmake:175 (message):
    Command failed: C:/Users/ILYVOI/Git/vcpkg/downloads/tools/cmake-3.12.4-windows/cmake-3.12.4-win32-x86/bin/cmake.exe;--build;.;--config;Debug;--target;install;--;/p:VCPkgLocalAppDataDisabled=true;/p:UseIntelMKL=No
    Working Directory: C:/Users/ILYVOI/Git/vcpkg/buildtrees/tiff/x86-uwp-dbg
    See logs for more information:
      C:\Users\ILYVOI\Git\vcpkg\buildtrees\tiff\install-x86-uwp-dbg-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_install_cmake.cmake:24 (vcpkg_build_cmake)
  ports/tiff/portfile.cmake:37 (vcpkg_install_cmake)
  scripts/ports.cmake:71 (include)


Error: Building package tiff:x86-uwp failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: tiff:x86-uwp
  Vcpkg version: 2018.10.20-nohash

Additionally, attach any relevant sections from the log files above.

C:\Users\ILYVOI\Git\vcpkg>.\vcpkg update
Using local portfile versions. To update the local portfiles, use `git pull`.
No packages need updating.

C:\Users\ILYVOI\Git\vcpkg>git pull
Already up to date.

[install-x86-uwp-dbg-out.log](https://github.com/Microsoft/vcpkg/files/2589420/install-x86-uwp-dbg-out.log)

--- a/ports/tiff/fix-stddef.patch
+++ b/ports/tiff/fix-stddef.patch
@@ -1,0 +1,13 @@
+diff --git a/port/lfind.c port/lfind.c
+index 087dfa9..20fa8b4 100644
+--- a/port/lfind.c
++++ b/port/lfind.c
+@@ -38,7 +38,7 @@
+ #ifdef _WIN32_WCE
+ # include <wce_types.h>
+ #else
+-# include <sys/types.h>
++# include <stddef.h>
+ #endif
+ 
+ #ifndef NULL

--- a/ports/tiff/portfile.cmake
+++ b/ports/tiff/portfile.cmake
@@ -15,6 +15,7 @@ vcpkg_extract_source_archive_ex(
         add-component-options.patch
         fix-cxx-shared-libs.patch
         crt-secure-no-deprecate.patch
+        fix-stddef.patch
 )
 
 if(VCPKG_TARGET_ARCHITECTURE STREQUAL "arm64" OR VCPKG_TARGET_ARCHITECTURE STREQUAL "arm")

